x/ibc/07-tendermint/evidence.go

```diff
@@ -93,19 +100,19 @@ func (ev Evidence) ValidateBasic() error {
 			sdkerrors.Wrap(err, "header 2 failed validation").Error(),
 		)
 	}
-	// Ensure that Heights are the same
-	if ev.Header1.Height != ev.Header2.Height {
-		return sdkerrors.Wrapf(clienttypes.ErrInvalidEvidence, "headers in evidence are on different heights (%d ≠ %d)", ev.Header1.Height, ev.Header2.Height)
+	// Header 2 must be at a latest height
+	if ev.Header2.Height <= ev.Header1.Height {
```

Comment:
I don't understand this. If we're talking about equivocation (DuplicateVoteEvidence), then ev.Header1 and ev.Header2 should be from the same height.